### PR TITLE
8.7 change entries

### PR DESCRIPTION
**Pull request by Zimmi48**
Unfinished please @herbelin @ppedrot help with mentioning #460 #558 #608 #612 #633 #676 #703 #714.

### CODE REVIEW

File: CHANGES [hunk 1]
```diff
@@ -46,19 +47,31 @@ Tactics
 - In "Tactic Notation" or "TACTIC EXTEND", entry "constr_with_bindings"
   now uses type classes and rejects terms with unresolved holes, like
   entry "constr" does. To get the former behavior use
```
Comment (psteckler) [inline CHANGES]: should this read "which entry constr allows" ?

File: CHANGES [hunk 2]
```diff
@@ -68,6 +81,9 @@ Standard Library
   and, consequently, choice of representatives in equivalence classes.
   Various proof-theoretic characterizations of choice over setoids in
   file ChoiceFacts.v.
+- New lemmas about iff and about orders on positive and Z.
+- New lemmas on powerRZ.
+- Strengthened statement of JMeq_eq_dep (closes BZ#4912).
 - The BigN, BigZ, BigZ libraries are not part anymore of Coq standard
```
Comment (psteckler) [inline CHANGES]: To me, that phrase sounds a bit awkward.

How about "are no longer part of the Coq standard ..." ?

File: CHANGES [hunk 3]
```diff
@@ -81,6 +97,9 @@ Standard Library
 
 Plugins
 
+- The Ssreflect plugin is now distributed with Coq. Its documentation has
+  been integrated as a chapter of the reference manual. This chapter is
+  still imperfect so feedback is welcome.
```
Comment (psteckler) [inline CHANGES]: How about "preliminary" instead of "imperfect" (which makes it sound like it's badly-written)

Comment (Zimmi48) [inline CHANGES]: I want to convey that it "has issues". Would "unpolished" sound good to you?

File: CHANGES (hunk 1 above)
Comment (Zimmi48) [inline CHANGES]: Well, I am not the author of these lines but I don't think so. As I understand it, what is meant is, on the contrary, that constr rejects terms with unresolved holes as well. Can someone confirm? @herbelin @ppedrot

Comment (psteckler) [inline CHANGES]: Oh, I thought the wording was meant to show a contrast. You're likely right.

File: CHANGES (hunk 3 above)
Comment (psteckler) [inline CHANGES]: That has the same flavor as "imperfect", to me. Actually, I'd say "preliminary", and omit "still". 

Or how about "work in progress"?

But not so important, in any case.

File: CHANGES [hunk 4]
```diff
@@ -46,19 +47,31 @@ Tactics
...
 
-- Goals context can be printed in a more compact way when "Set
-  Printing Compact Contexts" is activated.
+- Now supporting all kinds of binders, including 'pat, in syntax of record fields.
 
+Vernacular Commands
+
+- Goals context can be printed in a more compact way when `Set
+  Printing Compact Contexts` is activated.
+- Unfocussed goals can be printed with the `Set Printing Unfocused`
```
Comment (psteckler) [inline CHANGES]: "Unfocused" (single s), like the vernacular described here

File: CHANGES (hunk 3 above)
Comment (Zimmi48) [inline CHANGES]: I used "world in progress". Preliminary sounds like préliminaire and this word doesn't sound right in French.

**Review by herbelin — COMMENTED** [review]
#460: Maybe adding « The compatibility options for printing primitive projections ("Set Printing Primitive Projection Parameters" and "Set Printing Primitive Projection Compatibility") are now off by default. ».

#612: Maybe adding « New option "Set Ltac Batch Debug" on top of "Set Ltac Debug" for non-interactive Ltac debug output. » (if @JasonGross is ok).

#558: I would say nothing.

#663: Originally, I wanted to do more and seize the opportunity to improve the documentation of TACTIC EXTEND. But somehow, the changes of @roglo to camlp5 may make the hacks of #663 useless. So, let's wait first from the side of camlp5.

#703: It is a pity that no plugin has been made. So, maybe advertize the feature in the hope that someone do a plugin.

#714: Maybe say: « Added support for side effects hooks in "cbv", "cbn" and "simpl". The side effects are provided via a plugin "reduction-effects". » I have this plugin basically ready and I have to publish it. Please remind me if I forget.

For the rest, @ppedrot should know.

(edited for typos)

File: CHANGES (hunk 3 above)
Comment (psteckler) [inline CHANGES]: :+1:

**Comment by ppedrot** [comment]
#608: I wouldn't mention it, it is only relevant for Ltac2 and is internal fiddling to work around an incomplete API.

#676: "New variants of the first and solve tacticals that do not rely on parsing rules, meant to define tactic notations."

File: CHANGES [hunk 5]
```diff
@@ -156,7 +156,8 @@ Universes
 - Cumulative inductive types. see prefixes "Cumulative", "NonCumulative"
   for inductive definitions and the option "Set Inductive Cumulativity"
   in the reference manual.
-- New syntax `Type@{_}` for anonymous universes.
+- New syntax `Type@{_}` for anonymous universes (can serve to instantiate
+  explicitly a polymorphic definition with a flexible universe).
```
Comment (Zimmi48) [inline CHANGES]: @SkySkimmer Is that better? Maybe the mention of "flexible universe" is still too much and the CHANGES entry should stop at "definition"?

Comment (SkySkimmer) [inline CHANGES]: Maybe something like
> New syntax `foo@{_}` to instantiate a polymorphic definition with anonymous universes (can also be used with `Type`)

?

Comment (Zimmi48) [inline CHANGES]: Cool 👍

**Pull request merged**